PERP_model/funcs.py

```diff
@@ -0,0 +1,491 @@
+import numpy as np
```

Comment:
According to the standard Python style (https://www.python.org/dev/peps/pep-0008/) 
- Imports should usually be on separate lines
- Imports should be grouped in the following order:
    Standard library imports.
    Related third party imports.
    Local application/library specific imports.